ansible/roles/ansible-horizon-plugin/tasks/collect_compress.yml

```diff
@@ -1,15 +1,37 @@
 ---
-- debug:
-    msg: "{{ MANAGE_FILE }}"
 
 - name: Collect static files for horizon
-  command: "{{ MANAGE_FILE }}  collectstatic --noinput"
-  verbosity: 3
+  shell: "{{virtual_env}} && {{PYTHON_VERSION}} {{ MANAGE_FILE }}  collectstatic --noinput"
   notify:
      - restart webserver
 
 - name: compress the collected files
-  command: "{{ MANAGE_FILE }} compress --force"
-  verbosity: 3
+  shell: "{{virtual_env}} && nohup {{PYTHON_VERSION}} {{ MANAGE_FILE }} compress --force"
```

Comment:
if compress commands fail, by running it thru nohup, will the asnible also fail ?

Reply:
code changes done